src/FakeItEasy/Creation/ProxyOptions.cs

```diff
@@ -28,11 +27,7 @@ public void AddInterfaceToImplement(Type interfaceType)
 
             if (!interfaceType.GetTypeInfo().IsInterface)
             {
-                throw new ArgumentException(
-                    string.Format(
-                        CultureInfo.CurrentCulture,
-                        "The specified type '{0}' is not an interface",
-                        interfaceType.FullNameCSharp()));
+                throw new ArgumentException($"The specified type '{interfaceType.FullNameCSharp()}' is not an interface");

```

Comment:
Speaking of @thomaslevesque's `DoubleQuote`, perhaps a better reason to have a similar method would be to establish a consistent quoting style, rather than mechanism. There are about 10 places in the code where we use `'`s like this, instead of `"`s.
Although I think this could be a separate issue, if we care to pursue it.